spring-boot/src/main/java/org/springframework/boot/context/embedded/undertow/UndertowEmbeddedServletContainerFactory.java

```diff
@@ -345,6 +362,29 @@ private DeploymentManager createDeploymentManager(
 		return manager;
 	}
 
+	private void configureAccessLog(DeploymentInfo deploymentInfo) {
+		deploymentInfo.addInitialHandlerChainWrapper(new HandlerWrapper() {
+			@Override
+			public HttpHandler wrap(HttpHandler handler) {
+				try {
+					Xnio xnio = Xnio.getInstance(Undertow.class.getClassLoader());
+					XnioWorker worker = xnio.createWorker(OptionMap.builder().getMap());
+					File baseDir = (baseDirectory != null ? baseDirectory
```

Comment:
I think the same about logging into a temp dir, I did that just because Tomcat does too.
Anyhow I made those changes, but still recieving the AutoConfigure two tests fail.